lib/cocoapods/installer/xcode/pods_project_generator/target_installation_result.rb

```diff
@@ -73,9 +73,10 @@ def native_target_for_spec(spec)
           #         an array of all the test specs associated with this native target.
           #
           def test_specs_by_native_target
-            target.test_specs.group_by do |test_spec|
+            test_specs_by_native_target = target.test_specs.group_by do |test_spec|
               test_native_target_from_spec(test_spec)
             end
+            test_specs_by_native_target.delete_if { |k, _| k.nil? }
```

Comment:
this should have a test and also am error message that integrating test specs into pods that are not being built is not yet supported

Reply:
tests added